relay-event-normalization/src/normalize/mod.rs

```diff
@@ -1488,13 +1489,13 @@ mod tests {
 
         let proj = MeasurementsConfig {
             builtin_measurements: vec![foo.clone(), bar.clone()],
-            max_custom_measurements: 3,
+            max_custom_measurements: 4,
```

Comment:
yeah it also passes with 3.  
 
in this test, we insert 4 measurements, but the output should only be 3 because of duplicates. if we had the max custom measurements to 3, then we cant be sure if the output is 3 because we remove duplicates (as we wanna test) or because it removes excessive measurements (which is tested elsewhere), does this make sense?

Reply:
Seems reasonable.